Header header # ROS header

int32 num_sats

float32 mean_cn0
float32 median_cn0
float32 robust_mean_cn0
